lib/order_reporting.rb

```diff
@@ -1,12 +1,24 @@
 require 'order_reporting/engine'
 require 'order_reporting/report'
+require 'order_reporting/schedulers/delayed_job_scheduler'
+require 'order_reporting/queries/latest_orders_query'
```

Comment:
Yeah it's required as the class names don't match the folder structure. I did this intentionally and it's common amongst library code to do this. I categories the sub classes by folder but didn't want to add an additional namespace.